blocks/bricks/interfaces.py

```diff
@@ -141,31 +141,61 @@ class Initializable(RNGMixin, Brick):
 
     """
     has_biases = True
```

Comment:
I complained about the name `has_biases` in #697 and would be happy to see it go.